src/sentry/notifications/utils/participants.py

```diff
@@ -216,13 +218,38 @@ def get_send_to_owners(event: "Event", project: Project) -> Mapping[ExternalProv
     if user_ids_to_resolve:
         all_possible_users |= set(User.objects.filter(id__in=user_ids_to_resolve))
 
-    # Get all users in teams.
+    team_mapping = {ExternalProviders.SLACK: set()}
+    team_ids_to_remove = set()
     if team_ids_to_resolve:
+        # check for team Slack settings. if present, notify there instead
+        for team_id in team_ids_to_resolve:
+            team = Team.objects.get(id=team_id)
+            team_slack_settings = NotificationSetting.objects.get_settings(
```

Comment:
This should be using 
```python
NotificationSetting.objects.filter_to_subscribed_users(project, teams)
``` 
instead of 
```python
for team_id in team_ids_to_resolve:
  ...
  NotificationSetting.objects.get_settings(
    provider=ExternalProviders.SLACK,
    type=NotificationSettingTypes.ISSUE_ALERTS,
    team=team,
  )
```

Reply:
does that accept teams?

Reply:
also it's not about a user being subscribed at this point, I don't think a team can subscribe or unsubscribe?

Reply:
I think you're right, teams cannot fine tune.